app/controllers/mailgun_webhooks_controller.rb

```diff
@@ -26,6 +26,18 @@ def create_incoming_email
       received: params["Received"],
       attachment_count: params["attachment-count"],
     )
+
+    params.each_key do |key|
+      next unless /^attachment-\d+$/.match?(key)
+
+      contact_record.documents.attach(
+        io: params[key],
+        filename: params[key].original_filename,
+        content_type: params[key].content_type,
+        identify: false
```

Comment:
Curious about how we want to handle content sniffing.

Reply:
Maybe add a test for content type sad path?